lib/buttonLogic.js

```diff
@@ -0,0 +1,92 @@
+export const actions = {
+  markTaskAsCompleted: 'markTaskAsCompleted',
+  markTaskAsIncomplete: 'markTaskAsIncomplete',
+  stepCurrentTaskBackOnce: 'stepCurrentTaskBackOnce',
+  stepCurrentTaskForwardOnce: 'stepCurrentTaskForwardOnce',
+  fetchLatestWorkflowState: 'fetchLatestWorkflowState'
```

Comment:
`markTaskAsCompleted` endpoint

`markTaskAsIncomplete` endpoint

`stepCurrentTaskBackOnce` go through the breadcrumbs and fetch the previous task details

`stepCurrentTaskForwardOnce` go through the breadcrumbs and fetch the next task details

add `skipTask` endpoint

Reply:
Change the last test to calling the `skipTask` action

Reply:
rename `stepCurrentTaskBackOnce` to `getPreviousTask`
rename `stepCurrentTaskForwardOnce ` to `getNextTask`
add `skipTask`
TODO: figure out unskipping a task